### PR TITLE
Kyma dashboard

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- add job for pre and post main tests kyma-dashboard,
- add scripts for provision and start VM for kyma-dashboard tests,
- add scripts for cypress run.

**Related issue(s)**
https://github.com/kyma-project/kyma-dashboard/issues/34

### CODE REVIEW

**Comment by mrCherry97** [comment]
This is ready for review, but before merging still waiting for review: https://github.com/kyma-project/kyma-dashboard/pull/56

**Comment by mrCherry97** [comment]
PR for Kyma-dashboard is merged so this PR is ready for review and merge.

File: templates/data/kyma-dashboard-test.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                    - "jobConfig_default"
+                    - "request_small"
+                    - "vm_job_template_k3s"
+                    - "vm_job_labels_template"
+                    - "extra_refs_kyma-local"
+              # postsubmits
+              - jobConfig:
+                  name: "post-kyma-dashboard-test"
+                  optional: false
+                  run_if_changed: "/"
```
Comment (halamix2) [inline templates/data/kyma-dashboard-test.yaml:54]: You can replace `run_if_changed: "/"` with
```suggestion
                  always_run: true
```

**Review by halamix2 — CHANGES_REQUESTED** [review]

**Review by halamix2 — APPROVED** [review]

**Pull request merged**